tests/4.5/target_data/test_target_data_map_array_sections.c

```diff
@@ -1,6 +1,20 @@
-#include <omp.h>
```

Comment:
Might be good to keep this include, just to have access to the runtime functions.